lighthouse-core/lib/cdt/SDK.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const SDK = {
+  ...require('./generated/SourceMap.js'),
+};
+
+const originalMappings = SDK.TextSourceMap.prototype.mappings;
```

Comment:
add comment that this redefinition is done to avoid adding upperbound to array prototype